content/applications/services/helpdesk/overview/getting_started.rst

```diff
@@ -2,129 +2,164 @@
+   from public access to either private or company only access, portal users will be removed as
+   followers from both the team, and from individual tickets.
+
+Follow all team's tickets
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If a user should be notified about any updates regarding tickets for this team, select their name
+from the :guilabel:`Followers` drop-down. Multiple users can be selected to follow a single team.
+
+.. important::
+   External contacts can be selected in the :guilabel:`Followers` field. If the team's visibility is
+   set to :guilabel:`Invited internal users (private)`, followers will be notified about updates to
+   the team's tickets, but will not be able to view them in the portal.
+
 Automatically assign new tickets
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-When tickets are received, they will need to be assigned to a member of the support team. This can
-be done manually on each ticket individually, or through :guilabel:`Automatic Assignment`. Check the
-box next to :guilabel:`Automatic Assignment` to enable the feature for this team.
+When tickets are received, they need to be assigned to a member of the team. This is done either
+manually on each ticket individually, or through :guilabel:`Automatic Assignment`. Check the
+:guilabel:`Automatic Assignment` checkbox to enable this feature for the team.
 
-.. image:: getting_started/assignment-visibility.png
+.. image:: getting_started/helpdesk-visibility-assignment.png
    :align: center
-   :alt: View of a Helpdesk team settings page emphasizing the assignment and visibility features
+   :alt: View of a Helpdesk team settings page emphasizing the visibility and assignment features
          in Odoo Helpdesk
 
 Select one of the following assignment methods, based on how workload should be allocated across
 the team:
 
-- :guilabel:`Each user is assigned an equal number of tickets` assigns tickets to team members
-   based on total ticket count, regardless of the number of open or closed tickets they are
-   currently assigned.
-- :guilabel:`Each user has an equal number of open tickets` assigned tickets to team members based
-   on how many open tickets they are currently assigned. This option is useful for automatically
-   delegating a heavier workload to high-performers who tend to close tickets quickly.
+- :guilabel:`Each user is assigned an equal number of tickets` assigns tickets to team members based
+   on total ticket count, regardless of the number of open or closed tickets they are currently
+   assigned.
+- :guilabel:`Each user has an equal number of open tickets` assigns tickets to team members based on
+   how many open tickets they are currently assigned.
+
+.. tip::
+   If a team has high-performers who tend to close tickets quickly, selecting :guilabel:`Each user
+   has an equal number of open tickets` as the assignment method will ensure they have a heavier
+   workload.
```

Comment:
You are right,. I was attempting to outline a use case to help clarify the two options (because I think the way they are written in the UI can be a bit confusing).  I will change this up.